system-test/fixtures/sample/src/index.js

```diff
@@ -17,7 +17,7 @@
 // ** All changes to this file may be overwritten. **
 
 /* eslint-disable node/no-missing-require, no-unused-vars */
-const websecurityscanner = require('@google-cloud/web-security-scanner');
+const websecurityscanner = require('websecurityscanner');
```

Comment:
@sofisl looks like the same issue, we have a package name off somewhere leading to this